Sources/Orbit/Support/Layout/Strut.swift

```diff
@@ -5,7 +5,7 @@ import SwiftUI
 /// Used to fix layout issues where `frame(minHeight:)` causes issues.
 struct Strut: View {
     
-    let height: CGFloat
+    let height: CGFloat?
```

Comment:
What is this change related to?

Reply:
Oh I see... the module currently doesn't build. I wonder how we got into this situation given that each pull request builds the package...

Reply:
🤷